### PR TITLE
zh:add tibdmonitor remote write introduction

### PR DESCRIPTION
**Pull request by mikechengwei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

add tibdmonitor remote write introduction

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by mikechengwei** [comment]
@qiancai plz help me review this pr，thks.

File: zh/aggregate-multiple-cluster-monitor-data.md [hunk 1]
```diff
@@ -145,3 +143,38 @@ stringData:
...
+  clusters:
+  - name: basic
+  prometheus:
+    baseImage: prom/prometheus
+    version: v2.18.1
+    remoteWrite:
+      - url: "http://thanos-receive:19291/api/v1/receive"
+  grafana:
+    baseImage: grafana/grafana
+    version: 7.5.7
```
Comment (DanielZhangQD) [inline zh/aggregate-multiple-cluster-monitor-data.md:168]:
```suggestion
    version: 7.5.11
```

File: zh/aggregate-multiple-cluster-monitor-data.md [hunk 2]
```diff
@@ -145,3 +143,38 @@ stringData:
...
+apiVersion: pingcap.com/v1alpha1
+kind: TidbMonitor
+metadata:
+  name: basic
+spec:
+  clusters:
+  - name: basic
+  prometheus:
+    baseImage: prom/prometheus
+    version: v2.18.1
```
Comment (DanielZhangQD) [inline zh/aggregate-multiple-cluster-monitor-data.md:163]:
```suggestion
    version: v2.27.1
```

File: zh/aggregate-multiple-cluster-monitor-data.md [hunk 3]
```diff
@@ -145,3 +143,38 @@ stringData:
...
+metadata:
+  name: basic
+spec:
+  clusters:
+  - name: basic
+  prometheus:
+    baseImage: prom/prometheus
+    version: v2.27.1
+    remoteWrite:
+      - url: "http://thanos-receive:19291/api/v1/receive"
```
Comment (DanielZhangQD) [inline zh/aggregate-multiple-cluster-monitor-data.md:165]:
```suggestion
      - url: "http://thanos-receiver:19291/api/v1/receive"
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**